Index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>benjaminSemah.com</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a class="myLogo" href="#">My Logo</a>
+        </div>
+        <div class="navIcon">
+            <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
+        </div>
```

Comment:
- [ ] Good job in placing the hamburger button to the right. But we can try to replace the `div` tag with a `nav` tag.